### ISSUE TITLE
Get-PnPProvisioningTemplate - PageContents throws Object Reference Null error when page contains SPFX 1.7 Isolated Web Part

### ISSUE THREAD

**Issue by jimmywim** [SharePoint/PnP-Sites-Core#2017]
#### Category
[x] Bug
[ ] Enhancement

#### Environment
[x] Office 365 / SharePoint Online
[ ] SharePoint 2016
[ ] SharePoint 2013

#### Expected or Desired Behavior
PnP Provisioning Template generates with custom, domain isolated SPFX web parts in place.

#### Observed Behavior
If you have an SPFX 1.7 web part with `isDomainIsolated` set to `true`, the process fails with "Object Reference not set to an instance of an object"

#### Steps to Reproduce
Create an SPFX 1.7 web part solution that is has `isDomainIsolated` set to true in `package-solution.json`

Get a template using PowerShell or manually in C# using `web.GetProvisioningTemplate`

Specific error is at https://github.com/SharePoint/PnP-Sites-Core/blob/master/Core/OfficeDevPnP.Core/Pages/ClientSideWebPart.cs#L534

The `div` for the web part is missing the inner div that has the `data-sp-webpart` attribute on it. Indeed the isolated web part's markup seems a lot more lightweight. Looks like the provisioning engine needs a tweak to support isolated web parts.

**Comment by jansenbe** [comment]
Can confirm that the these webparts are indeed handled differently, an internal thread is ongoing to assess if this is the intended behaviour. For now there's a fix in in the pages API implementation that allows you to work with this web parts, do note that the provisioning engine does not yet support this. Support will be added based up on the feedback from the internal discussion.

**Comment by joakimhogberg** [comment]
I had this issue yesterday using only OOTB-webparts (Documents and List viewers). This morning I updated the SharePointPnPPowerShell modules and it works fine again!

**Comment by ruslanurban** [comment]
After installing `SharePointPnPPowerShellOnline` module version `3.7.1903.` I started receiving the following error:
``` error
Get-PnPProvisioningTemplate : The type of data at position 1142 is different than the one expected.
```
The error appears on sub-sites that have pages with SPFX web parts. Can be related to the original issue. Did not happen before installing latest version of the module. Installed versions: `3.7.1903.0`, `2.28.1807.0`, `2.23.1802.0`, `2.22.1801.0`, `2.20.1711.0`, `2.18.1709.1`.

PowerShell script:
``` powershell
            $web = Get-PnPWeb;

            Get-PnPProvisioningTemplate -Out $outputFilePath -Force `
                -ExcludeContentTypesFromSyndication:$ExcludeContentTypesFromSyndication `
                -PersistBrandingFiles `
                -PersistPublishingFiles `
                -IncludeSiteCollectionTermGroup `
                -IncludeTermGroupsSecurity `
                -TemplateDisplayName $web.Title `
                -IncludeSearchConfiguration `
                -IncludeNativePublishingFiles `
                -IncludeAllTermGroups `
                -IncludeSiteGroups `
                -Handlers All `
                -TemplateProperties $Properties;
```

**Comment by jansenbe** [comment]
can you share the output of get-pnpexception. Run this after you've received the error

**Comment by jimmywim** [comment]
Microsoft have changed something in CSOM I think, that's the second time today I've seen the error `The type of data at position...` - earlier was with calls to `Tenant.GetSitePropertiesByUrl` - https://twitter.com/jimmywim/status/1105459105462927360

**Comment by jimmywim** [comment]
@jansenbe Reproduced in another ticket, I've pasted the get-pnpexception: https://github.com/SharePoint/PnP-Sites-Core/issues/2103

**Comment by jimmywim** [comment]
This isn't happening with all SPO tenants, some work fine, other's don't.

**Comment by ruslanurban** [comment]
Have a look at the issues that seem to be related:
* [Get-PnPTenantSite throws error](https://github.com/SharePoint/PnP-Sites-Core/issues/2103)
* [Get-PnPProvisioningTemplate : The type of data at position 1144 is different than the one expected](https://github.com/SharePoint/PnP-Sites-Core/issues/2104)

The `Get-PnPException -All` did not return return anything. Here is the trace log:
``` text

PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:46.6557	[Extraction]	[17]	[Debug]	Code execution scope started	0ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:46.6567	[Extraction]	[17]	[Debug]	Base template available: BLANKINTERNET0template	0ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:46.6567	[Extraction]	[17]	[Debug]	ProgressDelegate registered	0ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:46.6567	[Extraction]	[17]	[Debug]	MessagesDelegate registered	0ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:46.6567	[Extraction]	[17]	[Debug]	IncludeAllTermGroups is set to true	1ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:46.6567	[Extraction]	[17]	[Debug]	PersistBrandingFiles is set to true	1ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:48.3985	[Regional Settings]	[17]	[Debug]	Code execution scope started	0ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:48.7041	[Regional Settings]	[17]	[Debug]	Code execution scope ended	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:48.7041	[Supported UI Languages]	[17]	[Debug]	Code execution scope started	0ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:49.0325	[Supported UI Languages]	[17]	[Debug]	Code execution scope ended	329ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:18:49.0335	[Term Groups]	[17]	[Debug]	Code execution scope started	0ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:01.5686	[Term Groups]	[17]	[Debug]	Code execution scope ended	12593ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:01.5686	[Fields]	[17]	[Debug]	Code execution scope started	0ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.0395	[Fields]	[17]	[Debug]	Context web is subweb, skipping site columns	472ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.0395	[Fields]	[17]	[Debug]	Code execution scope ended	472ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.0405	[Content Types]	[17]	[Debug]	Code execution scope started	0ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3422	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01002A2479FF33DD4BC3B1533A012B653717 - WorkflowServiceDefinition	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3422	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0100AA27A923036E459D9EF0D18BBD0B9587 - WorkflowServiceSubscription	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3422	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0100DC2417D125A4489CA59DCC70E3F152B2 - Rule	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3422	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01010000DEC92EFE5D445789D9FE4A3225A381 - Translation Status	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3422	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01010007FF3E057FA8AB4AA42FCB67B453FFC100E214EEE741181F4E9F7ACC43278EE8110003D357F861E29844953D5CAA1D4D8A3B - Html Page Layout	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3422	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0101002039C03B61C64EC4A04F5361F3851066 - Display Template	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3422	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0101002039C03B61C64EC4A04F5361F385106601 - Control Display Template	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0101002039C03B61C64EC4A04F5361F385106602 - Group Display Template	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0101002039C03B61C64EC4A04F5361F385106603 - Item Display Template	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0101002039C03B61C64EC4A04F5361F385106604 - Filter Display Template	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0101002039C03B61C64EC4A04F5361F385106605 - Display Template Code	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0101009148F5A04DDD49CBA7127AADA5FB792B - Rich Media Asset	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0101009148F5A04DDD49CBA7127AADA5FB792B00291D173ECE694D56B19D111489C4369D - Video Rendition	303ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0101009148F5A04DDD49CBA7127AADA5FB792B006973ACD696DC4858A76371B2FB2F439A - Audio	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0101009148F5A04DDD49CBA7127AADA5FB792B00AADE34325A8B49CDA8BB4DB53328F214 - Image	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x010100C5033D6CFB8447359FB795C8A73A2B19 - Design File	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x010100C568DB52D9D0A14D9B2FDCC96666E9F2007948130EC3DB064584E219954237AF3900B46186789C3140CC85BE610336E86BBB - Catalog-Item Reuse	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x010100E8711F0F931646FA949751442A907B22 - Translation Package	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x010100F8EF98760CBA4A94994F13BA881038FA - InfoPath Form Template	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x010106 - Master Page Preview	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0108003365C4474CAE8C42BCE396314E88E51F - Workflow Task (SharePoint 2013)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F93160 - SharePoint Server Workflow Task	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3426	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F93160000EEA96BE7C83464C9211129CD27F0409 - Collect Feedback Workflow Task (en-US)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F93160002E9DB43AC527439AB853AE4FC8360409 - Collect Signatures Workflow Task (en-US)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F93160003A15057E2AF34B67B32E14B94EB70409 - Approval Workflow Task (en-US)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0401 - Publishing Approval Workflow Task (ar-sa)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0402 - Publishing Approval Workflow Task (bg-bg)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0403 - Publishing Approval Workflow Task (ca-es)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0404 - Publishing Approval Workflow Task (zh-tw)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0405 - Publishing Approval Workflow Task (cs-cz)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0406 - Publishing Approval Workflow Task (da-dk)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0407 - Publishing Approval Workflow Task (de-de)	304ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0408 - Publishing Approval Workflow Task (el-gr)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0409 - Publishing Approval Workflow Task (en-US)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA040B - Publishing Approval Workflow Task (fi-fi)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA040C - Publishing Approval Workflow Task (fr-fr)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA040D - Publishing Approval Workflow Task (he-IL)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA040E - Publishing Approval Workflow Task (hu-HU)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0410 - Publishing Approval Workflow Task (it-it)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0411 - Publishing Approval Workflow Task (ja-jp)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0412 - Publishing Approval Workflow Task (ko-KR)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0413 - Publishing Approval Workflow Task (nl-nl)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0414 - Publishing Approval Workflow Task (nb-no)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0415 - Publishing Approval Workflow Task (pl-pl)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0416 - Publishing Approval Workflow Task (pt-br)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3436	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0418 - Publishing Approval Workflow Task (ro-RO)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0419 - Publishing Approval Workflow Task (ru-RU)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA041A - Publishing Approval Workflow Task (hr-hr)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA041B - Publishing Approval Workflow Task (sk-sk)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA041D - Publishing Approval Workflow Task (sv-se)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA041E - Publishing Approval Workflow Task (th-TH)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA041F - Publishing Approval Workflow Task (tr-tr)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0421 - Publishing Approval Workflow Task (id-id)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0422 - Publishing Approval Workflow Task (uk-ua)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0424 - Publishing Approval Workflow Task (sl-si)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0425 - Publishing Approval Workflow Task (et-EE)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0426 - Publishing Approval Workflow Task (lv-lv)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0427 - Publishing Approval Workflow Task (lt-lt)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA042D - Publishing Approval Workflow Task (eu-es)	305ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0439 - Publishing Approval Workflow Task (hi-in)	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA043F - Publishing Approval Workflow Task (kk-kz)	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0456 - Publishing Approval Workflow Task (gl-es)	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0804 - Publishing Approval Workflow Task (zh-CN)	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0816 - Publishing Approval Workflow Task (pt-PT)	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316000A245BAB39C6543159300E33084BA0C0A - Publishing Approval Workflow Task (es-ES)	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x01080100C9C9515DE4E24001905074F980F9316001 - Signatures Workflow Task Deprecated	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0110 - Post	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0120D5 - Document Collection Folder	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3446	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0120D520A8 - System Media Collection	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3455	[Content Types]	[17]	[Debug]	Adding content type to template: 0x0120D520A808 - Video	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3455	[Content Types]	[17]	[Debug]	Code execution scope ended	306ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:02.3455	[List instances]	[17]	[Debug]	Code execution scope started	0ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:06.5605	[List instances]	[17]	[Debug]	Code execution scope ended	4234ms	9622ab67-9819-4e96-99bf-1e6990602393
PowerShell_ISE.exe Information: 0 : 2019-03-13 08:19:06.5605	[Extraction]	[17]	[Debug]	Code execution scope ended	19999ms	9622ab67-9819-4e96-99bf-1e6990602393

```

**Comment by jansenbe** [comment]
The issue of @ruslanurban was a server side problem and has been fixed, the original issue with the isolated web parts was a bug as well and this was fixed with SPFX 1.8. Closing this issue now.
